src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.service;
 
+// import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
-     * Imagine that some user wants to login to your site.
+     * Imagine that some user wants to log into your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
+     * param email - user's email
+     * param password - user's password
+     * return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private static final UserService userService = new UserService();
```

Comment:
Could be not static variable
```suggestion
    private final UserService userService = new UserService();
```